apps/web/src/pages/v2/pair/[[...currency]].tsx

```diff
@@ -84,11 +84,16 @@ export default function PoolV2Page() {
                 <NextLinkFromReactRouter to={`/v2/add/${currencyIdA}/${currencyIdB}`}>
                   <Button width="100%">{t('Add')}</Button>
                 </NextLinkFromReactRouter>
-                <NextLinkFromReactRouter to={`/v2/remove/${currencyIdA}/${currencyIdB}`}>
-                  <Button ml="16px" variant="secondary" width="100%">
+                <NextLinkFromReactRouter to={`/v2/remove/${currencyIdA}/${currencyIdB}`} style={{ margin: '0px 8px' }}>
+                  <Button variant="secondary" width="100%">
                     {t('Remove')}
                   </Button>
                 </NextLinkFromReactRouter>
+                <NextLinkFromReactRouter to={`/v2/migrate/${pair?.liquidityToken?.address}`}>
+                  <Button variant="secondary" width="100%">
+                    {t('Migrate')}
```

Comment:
Lets hide it when has active v2 farm/ or less progressively show it when only farm v3